raiden/connection_manager.py

```diff
@@ -398,15 +398,16 @@ def _open_channels(self) -> bool:
             if channel_state not in funded_channels
         ]
         possible_new_partners = self._find_new_partners()
-        if possible_new_partners == 0:
+        if not possible_new_partners:
```

Comment:
Both here and below I would suggest changing the general "not" check to a more specific one.

So for this case `if len(possible_new_partners) == 0`

I have been bitten by unexpected "Falsy" values being evaluated by the broad `not` checks. Between explicitness and implicitness in coding I would almost always pick explicitness